install.sh

```diff
@@ -202,7 +172,8 @@ if { [[ -z "$home_max_size" ]] || [[ $home_max_size -eq 0 ]]; } &&
 
         # If valid choice, set max size variables
         if $valid_choice; then
-            IFS=' ' read -r home_max_size sgoinfre_max_size <<< "${campus_dict[$campus_name]}"
+            size_part=$(echo "${campus_name}" | cut -d ':' -f 2)
+            IFS=' ' read -r home_max_size sgoinfre_max_size _ <<< "$size_part"
```

Comment:
Oh I didn't know, is this a good practice to always have an extra variable for read to capture any unwanted remaining fields?